dev-aws/kafka-shared/iam/iam.tf

```diff
@@ -233,3 +233,28 @@ resource "kafka_topic" "iam_revoked_v1" {
     "cleanup.policy"    = "delete"
   }
 }
+
+
+resource "kafka_topic" "iam_password_change_forwarder_v1" {
+    name = "auth.iam-password-change-forwarder-v1"
+    replication_factor = 3
+    parititions = 1
+    config = {
+        # retain 100MB on each partition
+        "retention.bytes" = "104857600"
+        # keep data for 7 days
+        "retention.ms" = "604800000"
+        # allow max 1 MB for a message
+        "max.message.bytes" = "1048576"
+        "compression.type"  = "zstd"
+        "cleanup.policy"    = "delete"
+    }
+}
```

Comment:
I think we can use the [internal topic](https://github.com/utilitywarehouse/kafka-cluster-config/pull/202/files#diff-de69f762692bb71ed5f21bb386f47daa1aa19018d80b3bdfe72866dc22773ec3R31-R45) internal topic as the blueprint fro the public one.